roerich/algorithms/calc_metrics.py

```diff
@@ -0,0 +1,68 @@
+from .cpdc import ChangePointDetectionBase
```

Comment:
@maxslamdunk, move this file into https://github.com/HSE-LAMBDA/roerich/tree/main/roerich/change_point folder, please. And remane it as windows.py, for an example.